pkg/ddl/ingest/checkpoint.go

```diff
@@ -348,7 +362,7 @@ func (s *CheckpointManager) updateCheckpoint() error {
 	defer s.sessPool.Put(sessCtx)
 	ddlSess := sess.NewSession(sessCtx)
 	err = ddlSess.RunInTxn(func(se *sess.Session) error {
-		template := "update mysql.tidb_ddl_reorg set reorg_meta = %s where job_id = %d and ele_id = %d and ele_type = %s;"
+		template := "update mysql.tidb_ddl_reorg set reorg_meta = %s where job_id = %d and ele_type = %s;"
```

Comment:
Then for all different indexes, the reorg_meta are the same?
Is it expected?

Reply:
Yes. The checkpoint only records table row handles.